core/src/i18n/en.yaml

```diff
@@ -198,6 +201,7 @@ certificates:
     with-csr: Enable to use a Certificate Signing Request instead of providing domain names manually.
   with-csr: Use a CSR
 cluster-addons:
+  description: <0>Cluster Addons Configuration</0> defines cluster-wide addons that provide all information required to convert Helm charts into Service Classes in Service Catalog.
```

Comment:
Aaaah I see, you went for consistent sentence structure here and in line 3, makes sense.